api/http/management_filetransfer.go

```diff
@@ -361,7 +362,7 @@ func (h ManagementController) downloadFileResponse(c *gin.Context, params *fileT
 func (h ManagementController) downloadFileResponseProcessMessage(c *gin.Context,
 	params *fileTransferParams, request *model.DownloadFileRequest, wsMessage *natsio.Msg,
 	deviceTopic string, latestOffset *int64, numberOfChunks *int, responseHeaderSent *bool,
-	ticker *time.Ticker) error {
+	fileInfo *wsft.FileInfo, ticker *time.Ticker) error {
```

Comment:
This argument looks excessive. Why not just declare the variable on the next line?

Reply:
sorry I merged before reading that. I had something in mind while doing that. if I do not find it I will fix with the next patch I do here.